docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Azure
+description: Instructions to infer an ONNX model remotely with an Azure endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Azure-ExecutionProvider
+---
+
+# Azure Execution Provider (Preview)
```

Comment:
Do we want to call this document Azure Execution Provider? I'm thinking about what people will search for to find it. Would Cloud execution provider make more sense?

Reply:
@jeffmend specifically wanted to call this Azure EP. We'd called it Cloud EP before :-).

Reply:
Yes, this was an intentional choice to go with Azure EP in the docs.  would be good if it came up in searches.

Reply:
Ok, perhaps we should also (not in this PR, but later) add a tutorial that uses more scenario specific terminology and that could link to this doc